### ISSUE TITLE
Setup ucontex clientside application

### ISSUE THREAD

**Issue by joshikeno** [ucontex/ucontex#1]
- [ ] setup next custom server
- [ ] setup graphql
- [ ] setup styled-components
- [ ] formik

@core

**Comment by joshikeno** [comment]
@sparkydman any updates?

**Comment by sparkydman** [comment]
Yes. But am having problem with the font. Am trying to load font locally using @local_font I couldn't. I have tried creating static or public folder still no difference. Tried some suggestion by configuring next.config.js file and still no change.

Where I am now am trying to copy all master to a prod branch that I want to deploy, since is continuous deployment I wouldn't want it to automatically deploy unsuitable branch.

**Comment by joshikeno** [comment]
@sparkydman rather you add font via the @font-face CSS feature in the HTML page.

**Comment by sparkydman** [comment]
Thank you 
That was what I was doing. Sorry I meant to say @font-face instead. I will try every suggestion later
